source/includes/inkling-reference/_schema.html.md

```diff
@@ -118,6 +122,7 @@ end
 
 Select the Inkling tab to show a schema that has a field with a primitive type and a field with a structured type.
 
+#link
```

Comment:
This would create a new heading called "link" which isn't what you want I don't think. So go ahead and remove this.